tokio/src/sync/watch.rs

```diff
@@ -669,6 +669,22 @@ impl<T> Receiver<T> {
         self.version.decrement();
     }
 
+    /// Marks the state as unchanged.
+    ///
+    /// The current value can will be considered seen by the receiver.
+    ///
+    /// After invoking this method [`has_changed()`](Self::has_changed)
+    /// *MAY* return `false` and [`changed()`](Self::changed) *MAY* not return
+    /// immediately. This is because a new value could be sent before
+    /// this method returns.
```

Comment:
This is only the case if a value could be sent concurrently with this call. If there are no concurrent sends, then this does not apply.

Reply:
Yeah I'll explain that better. Should I also provide a warning that the value being marked as "seen" could change if used concurrently as well? e.g.

1. receiver version is 2, shared version is 4. `mark_unchanged` is called
2. another task sends a message so shared version becomes 6, `mark_unchanged` sets receiver version to 6
3. the caller intended to ignore the value associated with version 4 but instead skips 2 values